docs/en/guides/core_compatibility.rst

```diff
@@ -0,0 +1,44 @@
+Compatibility Guide for ESP32 Arduino Core
+====================================================
```

Comment:
Remove extra chars.

Reply:
Done, btw. @pedrominatel why this did not trigger any warning or error?